money.go

```diff
@@ -85,6 +88,33 @@ func New(amount int64, code string) *Money {
 	}
 }
 
+// NewFromFloat creates and returns new instance of Money from a float64.
+func NewFromFloat(amount float64, currency string) *Money {
+	currencyDecimals := math.Pow(10, float64(GetCurrency(currency).Fraction))
```

Comment:
Pow10 is so much more optimised and you don't need casting.
```suggestion
	currencyDecimals := math.Pow10(GetCurrency(currency).Fraction)
```